### DISCUSSION TITLE
Update Positron IDE in Ubuntu

### DISCUSSION THREAD

**Discussion by marcozanotti** [posit-dev/positron#5578] [Q&A] [answered]
How one can update the Positron IDE in Ubuntu?

**Comment by jmcphers** [comment, marked as answer]
The easiest way right now is to download the new `.deb` file and install it. You can get a new `.deb` from our releases page:

https://github.com/posit-dev/positron/releases

Once you have it downloaded, double-clicking on it should open it in your package manager and offer to update. If that doesn't work, you can also use the command line to install the file directly, e.g.:

```bash
sudo dpkg -i Positron-2024.12.0-80.deb
```

We're working on a more automated update method.

**Reply by marcozanotti** [reply]
I see. Thank you for your help!
